server/setting.go

```diff
@@ -64,23 +63,23 @@ func (s *Server) registerSettingRoutes(g *echo.Group) {
 		if settingPatch.Name == api.SettingAppIM {
 			var value api.SettingAppIMValue
 			if err := json.Unmarshal([]byte(settingPatch.Value), &value); err != nil {
-				return err
+				return echo.NewHTTPError(http.StatusBadRequest, "Malformed setting value for IM App").SetInternal(err)
```

Comment:
Remove "App". 

1. Be consistent with other logs
2. App is sort of an implementation detail (unless we do expose it as IM application setting instead of just IM setting